app/views/declaration/destinationCountries/routing_countries_summary.scala.html

```diff
@@ -19,26 +19,44 @@
 @import forms.declaration.RoutingQuestionYesNo.{no, RoutingQuestionPage, yes}
 @import models.requests.JourneyRequest
 @import services.model.Country
-@import views.components.inputs.RadioOption
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukButton
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukRadios
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukDetails
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukSummaryList
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukFieldset
```

Comment:
Do you need to import every component when you're importing everything in line 26?

Reply:
I have addressed this.
I like to specify which ones I have used, but I am going to follow our existing approach